onnxruntime/python/tools/tensorrt/perf/benchmark_wrapper.py

```diff
@@ -27,10 +28,11 @@ def main():
 
     model_to_fail_ep = {}
 
-    benchmark_fail_csv = 'fail.csv' 
-    benchmark_metrics_csv = 'metrics.csv'
-    benchmark_success_csv = 'success.csv' 
-    benchmark_latency_csv = 'latency.csv'
+    commit = get_latest_commit_hash()
+    benchmark_fail_csv = commit + '_fail.csv' 
```

Comment:
Excel doesn't allow you to open two files with the same name. I found when manually comparing (*script to replace manual comparison to come*), it was difficult to keep track of which result was new/ old. While the commit number uniquely identifies the csv's, it does make the filename quite long. Any suggestions?

Reply:
look into writing the shortened form of the SHA1 hash rather than the full commit id if you're concerned about filename length.